app/styles/datepicker.scss

```diff
@@ -0,0 +1,44 @@
+.datepicker {
+    
+    width:300px;
+   // line-height: 30px;
+    font-weight: 300;
+    content:none;
+    
+
+
+}
+
+ tr,td,th {
```

Comment:
tr, td are very common to be used all over the product. So, it's always recommended to wirte component specific css with proper classnames hierarchy.

Reply:
Done.